src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -837,6 +854,20 @@ public function findDefinition($id)
     }
 
     /**
+     * Returns the case full id used at registration time if any.
+     *
+     * @param string $id
+     *
+     * @return string
+     */
+    public function getCaseFullId($id)
```

Comment:
Yeah, if anything it should be `casefullId` (or `getCasefullId`). But I like @theofidry's `caseSensitiveId` more 👍